CHANGELOG.md

```diff
@@ -1,16 +1,22 @@
 ## vNext (TBD)
 
+### Breaking Changes
+* Renamed `SyncErrorCategory.resolve` category to `SyncErrorCategory.webSocket`, `SyncResolveError` to `SyncWebSocketError` and `SyncResolveErrorCode` to `SyncWebSocketErrorCode`. Added new codes to  `SyncWebSocketErrorCode`.
+
 ### Enhancements
 * None
 
 ### Fixed
-* None
+* Fixed performance degradation on SubQueries (Core upgrade).
+* Fixed several cases where wrong type of exception was thrown (Core upgrade).
```

Comment:
Is this fixing the switch user exception type? 
We are missing a mention for the switch user exception type fix. I suggest we add it explicitly here.

Reply:
Added.